From e60d1cafda0078354c7d12bace20e92cf6c78f69 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 29 Jun 2016 19:15:07 +0900
Subject: [PATCH 1786/2066] arm64: dts: r8a7795-salvator-xs: Add AUDIO-CLKOUTn
 synchronous with L/R clock

This patch adds clkout-lr-synchronous property which synchronizes
L/R clock to the rcar_sound node.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 .../arm64/boot/dts/renesas/r8a7795-salvator-xs.dts |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
index 11db0ab..5acf75a 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
@@ -601,6 +601,7 @@
 	/* audio_clkout0/1/2/3 */
 	#clock-cells = <1>;
 	clock-frequency = <11289600>;
+	clkout-lr-synchronous;
 
 	status = "okay";
 
-- 
1.7.5.4

